docs/_functions/global/FETCH.md

```diff
@@ -3,6 +3,8 @@ name: FETCH
 parameters:
   - url
   - args
+ts_ignore: true
+# Make sure to update fetch.d.ts if changing the docs below!
```

Comment:
@j-f1 I've made some changes from https://github.com/StackExchange/dnscontrol/pull/1809, how do I generate the new `fetch.d.ts? `go generate` only generates `types/dnscontrol.d.ts`.

Reply:
Copy+paste, unfortunately.

Reply:
Perhaps it is a good idea to ensure this necessary step in the docs? https://docs.dnscontrol.org/release/release-engineering#step-1.-rebuild-generated-files

Reply:
What do you mean? Unless someone changes the `FETCH` function’s interface without updating the docs (which is unlikely), they’ll see this message.

Reply:
Thanks for your comment. I'll further explain my comment with a code example. From https://github.com/StackExchange/dnscontrol/commit/55889582aeddc92d424a50f4577be030dfd04f49#diff-bda8825028c14bd7feb8eb5463e708023019b3ff7f710af883e4b63eba0b9ec3R12 I had to change the URLs. By copy/paste `global/FETCH.md` it's generates this diff which (*tested with JetBrains IDE*) is not valid.

```diff
- Compared to `fetch` from Fetch API, `FETCH` will call [PANIC](https://dnscontrol.org/js#PANIC) to terminate the execution of the script, and therefore DNSControl, if a network error occurs.
+ Compared to `fetch` from Fetch API, `FETCH` will call [PANIC](PANIC.md) to terminate the execution of the script, and therefore DNSControl, if a network error occurs.
```

This should be (*I guess*) a full URL reference?

```diff
- Compared to `fetch` from Fetch API, `FETCH` will call [PANIC](https://dnscontrol.org/js#PANIC) to terminate the execution of the script, and therefore DNSControl, if a network error occurs.
+ Compared to `fetch` from Fetch API, `FETCH` will call [PANIC](https://docs.dnscontrol.org/language-reference/top-level-functions/panic) to terminate the execution of the script, and therefore DNSControl, if a network error occurs.
```

I suspect you corrected this URL once? So the previous answer 'copy/paste' doesn't seem to be entirely valid. Or maybe I'm overlooking something?

Reply:
Ah yeah, great point. I must have changed those URLs, then. I’m not convinced it’s worth the effort to thoroughly document this since you’re likely to be the last person to touch the file (and especially the links) for a long time.